packages/web/src/scss/components/Card/README.md

```diff
@@ -0,0 +1,398 @@
+
+```html
+<div class="CardEyebrow">Content options</div>
+<h4 class="CardTitle">Card Title</h4>
+```
+
+## CardFooter
+
+Use CardFooter for actions or any other content at the bottom of the Card. When using Cards with CardFooter in a Grid,
+the CardFooters will automatically line up.
+
+```html
+<footer class="CardFooter">
+  <!-- … -->
+</footer>
+```
+
+### Footer Alignment
+
+The footer can be horizontally aligned to the start, center, or end of the Card. To align the footer, use one of the
+following CSS modifiers:
+
+- `CardFooter--alignmentXLeft`
+- `CardFooter--alignmentXCenter`
+- `CardFooter--alignmentXRight`
+
+## Full Example
+
+When you put it all together:
+
+```html
+<article class="Card Card--vertical Card--boxed">
+  <div class="CardMedia CardMedia--auto">
+    <div class="CardMedia__canvas">
+      <img src="…" alt="" />
+    </div>
+  </div>
+  <div class="CardLogo">
+    <div class="PartnerLogo PartnerLogo--medium PartnerLogo--safeArea">
+      <img src="…" alt="Logo" />
+    </div>
+  </div>
+  <div class="CardBody">
+    <div class="CardEyebrow">Content options</div>
+    <h4 class="CardTitle CardTitle--heading">
+      <a href="#" class="CardLink">Card Title</a>
+    </h4>
+    <p>Card content</p>
+  </div>
+  <footer class="CardFooter CardFooter--alignmentXLeft">
+    <a href="#" class="Button Button--medium Button--primary">Primary</a>
+    <a href="#" class="Button Button--medium Button--secondary">Secondary</a>
+  </footer>
+</article>
+```
+
+## Card Grid
+
+In a typical use case, you will display multiple Cards in a [Grid][grid].
+
+```html
+<div class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-3">
+  <article class="Card Card--vertical">
+    <!-- … -->
+  </article>
+  <article class="Card Card--vertical">
+    <!-- … -->
+  </article>
+  <article class="Card Card--vertical">
+    <!-- … -->
+  </article>
+</div>
+```
+
+Depending on your situation, you may want to use the list semantics. And it will work!
+
+```html
+<ul class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-3">
+  <li class="Card Card--vertical">
+    <!-- … -->
+  </li>
+  <li class="Card Card--vertical">
+    <!-- … -->
+  </li>
+  <li class="Card Card--vertical">
+    <!-- … -->
+  </li>
+</ul>
+```
+
+## Best Practices
+
+### Making the Whole Card Clickable
+
+To make the whole Card clickable, use the provided CardLink subcomponent. For best accessibility, you would typically
+wrap your CardTitle text in the CardLink component:
+
+```html
+<h4 class="CardTitle">
+  <a href="#" class="CardLink">Card title</a>
+</h4>
+```
+
+This establishes a clickable overlay over the whole Card, making it easier for users to interact with the Card.
+
+ℹ️ Don't worry, any interactive elements inside the Card (like links or buttons) will still work as expected.
+
+ℹ️ The text content of CardBody remains selectable and copyable even when the whole Card is clickable.
```

Comment:
i wonder if we will have to do this optional in the future

Reply:
Do we have any source that can be linked to this? I mean it like it will more explain why it is done this way.

Reply:
I added a modifier for this: `.CardBody--selectable` which will become `isSelectable` in React.